qlib/contrib/model/stacking.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from email.mime import base
```

Comment:
What's the necessity to import it?

Reply:
Removed. This should be auto imported when typing base_estimator.